tests/integration/IntegrationTest.php

```diff
@@ -113,7 +113,31 @@ public function testDeleteUsers()
             if ($response["status"] == "completed") {
                 return;
             }
-            sleep(1);
+            sleep(0.5);
```

Comment:
`sleep` only works with int. Let's use `usleep` currently it sleeps zero seconds

https://www.php.net/manual/en/function.usleep.php

Reply:
yes, wanted to make it faster but didn't work